freeipa/src/test/java/com/sequenceiq/freeipa/converter/telemetry/TelemetryConverterTest.java

```diff
@@ -0,0 +1,66 @@
+package com.sequenceiq.freeipa.converter.telemetry;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNull;
+
+import org.junit.Before;
+import org.junit.Test;
```

Comment:
Could you please use JUnit 5 and assertJ (`Assertions.assertThat()`) if possible?